lib/fontist/utils/ui.rb

```diff
@@ -37,6 +37,9 @@ def self.say(message)
 
       def self.ask(message, options = {})
         new.ask(message, options)
+      rescue Errno::EBADF
+        say(" ... cannot get the answer in detached mode.")
```

Comment:
"ERROR: Fontist is unable to obtain agreement without an interactive prompt.\n Please provide explicit agreement at execution or re-run Fontist with an interactive prompt."

Reply:
This will raise a `Fontist::LicensingError`?

Reply:
Yes, it will

```
 it "raises licensing error in fully detached mode" do
        example_formula("andale.yml")
        stub_license_agreement_prompt_with_exception
        expect { Fontist::Font.install("andale mono") }.to raise_error(
          Fontist::Errors::LicensingError,
        )
end
```